doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -276,17 +276,11 @@ jq
 Installing jq
 =============
 
-``jq`` is a command line tool for parsing JSON output that is used by the Dataverse Software installation script. It is available in the EPEL repository::
+``jq`` is a command line tool for parsing JSON output that is used by the Dataverse Software installation script. It is available in the ``appstream`` repository::
```

Comment:
Appstream means it's just enabled, right?

Reply:
correct.